src/docs/howto-listing/extend-listing-data-in-ftw/index.md

```diff
@@ -54,9 +55,11 @@ for adding extended data directly to a listing. You will also need to
 make custom changes to your listing page, if you want to show the
```

Comment:
These instructions no longer exist, so we could maybe go with "For a more complex attribute, you will need to add extended data..." i.e. omit the mention of instructions.